Dockerfile

```diff
@@ -32,23 +32,11 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM buildpack-deps:stretch AS base
+FROM golang:1.9.4 AS base
 # allow replacing httpredir or deb mirror
 ARG APT_MIRROR=deb.debian.org
 RUN sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
 
-
-FROM base AS golang
-# IMPORTANT: If the version of Go is updated, the Windows to Linux CI machines
-#            will need updating, to avoid errors. Ping #docker-maintainers on IRC
-#            with a heads-up.
-# IMPORTANT: When updating this please note that stdlib archive/tar pkg is vendored
-ENV GO_VERSION 1.9.4
-RUN curl -fsSL "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" \
-	| tar -xzC /usr/local
-ENV PATH=/usr/local/go/bin:/go/bin:$PATH GOPATH=/go
```

Comment:
Guess it doesn't make a difference for us, but the order is slightly different in the official Golang images (just leaving it here);

```Dockerfile
ENV GOPATH /go
ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
```